src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/LaunchDarklyConfiguration.java

```diff
@@ -35,5 +35,7 @@ public void addInterceptors(InterceptorRegistry registry) {
                 .addPathPatterns("/refdata/external/v1/organisations/pba");
         registry.addInterceptor(featureConditionEvaluation)
             .addPathPatterns("/refdata/internal/v1/organisations/{orgId}/mfa");
+        registry.addInterceptor(featureConditionEvaluation)
```

Comment:
Please remove the lines 38 and 39 as the endpoint path pattern has already been added in lines 34 and 35